app/assets/javascripts/admin/addon/components/admin-config-areas/emojis-settings.gjs

```diff
@@ -0,0 +1,54 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import didInsert from "@ember/render-modifiers/modifiers/did-insert";
+import { service } from "@ember/service";
+import DBreadcrumbsItem from "discourse/components/d-breadcrumbs-item";
+import { ajax } from "discourse/lib/ajax";
+import i18n from "discourse-common/helpers/i18n";
+import { bind } from "discourse-common/utils/decorators";
+import AdminFilteredSiteSettings from "admin/components/admin-filtered-site-settings";
+import SiteSetting from "admin/models/site-setting";
+
+export default class AdminConfigAreasEmojisSettings extends Component {
```

Comment:
I see this is almost identical to the flags one, I thought this may happen, I think we may eventually want to extract these out into an `AdminConfigAreaSettings` component which can tag an `@area` or `@category` filter. But for now this is good :+1:

Reply:
Had the exact same thought. I guess when we collect enough of them we will have enough information to make a general purpose one. 👍